### PR TITLE
Create UI for Eventing Module

### PR DESCRIPTION
**Pull request by grischperl**
**Description**

Changes proposed in this pull request:

- Create UI for Eventing Module

**Related issue(s)**

#120

### CODE REVIEW

File: config/ui-extensions/eventing/general [hunk 1]
```diff
@@ -0,0 +1,15 @@
+resource:
+  kind: Eventing
+  group: operator.kyma-project.io
```
Comment (k15r) [inline config/ui-extensions/eventing/general:3]: please change the name

File: config/default/kustomization.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ resources:
 - ../manager
 - ../webhook
 - ../busola
+- ../ui-extensions
```
Comment (k15r) [inline config/default/kustomization.yaml:23]: we have two folders containing configuration for busola:
- busola 
- ui-extensions

Please consolidate into one

File: config/ui-extensions/eventing/general (hunk 1 above)
Comment (muralov) [reply config/ui-extensions/eventing/general:3]: @k15r [Telemetry](https://github.com/kyma-project/telemetry-manager/blob/3ed3e38fc12408347388d3355e4ee338fa836485/config/busola/telemetry_busola_extension_cm.yaml#L103C38-L103C38) and [KEDA](https://github.com/kyma-project/keda-manager/blob/0f0989db147114a68f9db2fba534977451d1946e/config/ui-extensions/kedas/general#L3C5-L3C5) modules seem to have the same with ours. 
What value did you want and why do you want it?

File: config/default/kustomization.yaml (hunk 2 above)
Comment (muralov) [reply config/default/kustomization.yaml:23]: KEDA uses `ui-extensions`, Telemetry uses `busola`. We may use `ui-extensions` as it seembs better structured, we need to convert exising `busola` in EM. NATS manager already has `ui-extensions`.

File: config/ui-extensions/eventing/general (hunk 1 above)
Comment (k15r) [reply config/ui-extensions/eventing/general:3]: Sorry, wrong line :)

File: config/ui-extensions/eventing/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+configMapGenerator:
+  - name: operator.kyma-project.io
```
Comment (k15r) [inline config/ui-extensions/eventing/kustomization.yaml:2]: This name needs to be changed

Comment (muralov) [reply config/ui-extensions/eventing/kustomization.yaml:2]: yes, it needs to be `eventing.operator.kyma-project.io`.

**Review by k15r — APPROVED** [review]

**Review by k15r — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Pull request merged**